src/Code.js

```diff
@@ -58,27 +38,47 @@ function setApiKey() {
   const button = result.getSelectedButton();
   const text = result.getResponseText();
   if (button === ui.Button.OK) {
-    setProperty(PROP_MB_API_KEY, text);
-    ui.alert(`API key is ${text}.`);
+    try {
+      setProperty(PROP_MB_API_KEY, text);
+      ui.alert(`API key is ${text}`);
+    } catch (e) {
+      ui.alert(e.message);
+    }
   }
 }
 
 function deleteAllSettings() {
-  const properties = PropertiesService.getDocumentProperties();
-  if (properties) {
-    properties.deleteAllProperties();
-  } else {
-    testProperties = {};
+  const ui = SpreadsheetApp.getUi();
+  const result = ui.alert(
+    'Are you sure to delete all settings?',
+    ui.ButtonSet.YES_NO,
+  );
+
+  if (result === ui.Button.YES) {
+    deleteAllProperties();
+    ui.alert('Deployment ID and API key have been deleted');
```

Comment:
Is there a reason to have another dialogue popup for this (and other) menu items? Perhaps a popup only in the case of failure?

Reply:
It is to let users know what they have done actually but it is ok to remove it when you feel not so useful :)

Reply:
Ok got it. Let's remove it across the add-on.